ppb/systems/renderer.py

```diff
@@ -111,12 +113,14 @@ def __init__(
         resolution=DEFAULT_RESOLUTION,
         window_title: str = "PursuedPyBear",
         target_frame_rate: int = 30,
+        target_camera_width=25,
```

Comment:
This is the default scene width? This is significantly larger than before, right?

Reply:
Flipped a coin between a 32 pixel default sprite and a 64. I'm willing to switch it.

Reply:
(But yes, this is 25 x ~19, previously, we were 10 x ~7.5)